packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -36,7 +36,8 @@ interface SelectableCollectionOptions {
   keyboardDelegate: KeyboardDelegate,
   autoFocus?: boolean,
   focusStrategy?: FocusStrategy,
-  wrapAround?: boolean
+  wrapAround?: boolean,
+  escapeClearsSelection?: boolean
```

Comment:
Can we combine this with #225 so it handles mouse support as well? I'd like to have one prop to disable empty selection as opposed to a specific one just for the escape key. Maybe `disallowEmptySelection`? 🤔 

No need to actually combine the PRs, just make the name consistent.

Reply:
sure!